keycloak_client.go

```diff
@@ -81,12 +80,22 @@ func New(config Config) (*Client, error) {
 		errTolerance = time.Minute
 	}
 
+	var httpClient = gentleman.New()
```

Comment:
Why did this part of code move? 
Just out of curiosity

Reply:
It was just in order to keep the declaration of httpClient and its configuration close to each other